### PR TITLE
Update PageObjects.md

### PR DESCRIPTION
**Pull request by PetrKnedlik**
changed page to page.js so it's more clear to everyone

## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by PetrKnedlik** [comment]
Small change in documentation

File: docs/PageObjects.md [hunk 1]
```diff
@@ -15,9 +15,11 @@ The goal of using page objects is to abstract any page information away from the
 
 ## Making A Page Object
 
-First off, we need a main page object that we call `Page`. It will contain general selectors or methods which all page objects will inherit from.
+First off, we need a main page object that we call `Page.js`. It will contain general selectors or methods which all page objects will inherit from.
 
 ```js
+
```
Comment (christian-bromann) [inline docs/PageObjects.md:21]: this empty line seems unnecessary

```suggestion
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]
One comment otherwise LGTM

**Comment by PetrKnedlik** [comment]
@christian-bromann removed

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**